src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1045,12 +1057,60 @@
   <data name="General_VersionLastChecked.Text" xml:space="preserve">
     <value>Last checked: </value>
   </data>
+  <data name="General_LastSettingsBackupDate.Text" xml:space="preserve">
+    <value>Last backup at:</value>
+  </data>
+  <data name="General_SetupAndRestoreLocationText.Text" xml:space="preserve">
+    <value>Backup and restore location</value>
+  </data>
+  <data name="General_SetupBackupAndRestoreLocationText.Header" xml:space="preserve">
+    <value>Location</value>
+  </data>
+  <data name="General_SetupBackupAndRestoreLocationText.Description" xml:space="preserve">
+    <value>Select a location to store the backup.</value>
```

Comment:
Is this description needed? Backup-file location as a header should suffice and keeps things concise and clear?